tests/di/container_cache_directory_test.php

```diff
@@ -0,0 +1,122 @@
+<?php
+/**
+*
+* This file is part of the phpBB Forum Software package.
+*
+* @copyright (c) phpBB Limited <https://www.phpbb.com>
+* @license GNU General Public License, version 2 (GPL-2.0)
+*
+* For full copyright and license information, please see
+* the docs/CREDITS.txt file.
+*
+*/
+
+namespace
+{
+	class container_cache_directory_test extends \phpbb_test_case //phpbb_di_container_test
+	{
+		protected $config_php;
+
+		/**
+		* @var \phpbb\di\container_builder
+		*/
+		protected $builder;
+		protected $phpbb_root_path;
+		protected $filename;
+
+		public function setUp(): void
```

Comment:
Your PR is against 3.2.x and hence should maintain compatibility with PHP < 7.
```suggestion
		public function setUp()
```